tests/lib/testers/rule-tester.js

```diff
@@ -621,9 +612,33 @@ describe("RuleTester", () => {
 
     });
 
+    it("throw an error when an unknown config option is included", () => {
+
+        assert.throws(() => {
+            ruleTester.run("no-eval", require("../../fixtures/testers/rule-tester/no-eval"), {
+                valid: [
+                    { code: "Eval(foo)", foo: "bar" }
+                ],
+                invalid: []
+            });
+        }, /ESLint configuration is invalid./);
```

Comment:
Can this message be improved for RuleTester tests? (Maybe instead of "ESLint configuration", something like "Test configuration"... maybe?)